runtimes/server-interface/workspace.ts

```diff
@@ -19,6 +19,7 @@ export type Workspace = {
     getTextDocument: (uri: string) => Promise<TextDocument | undefined>
     getAllTextDocuments: () => Promise<TextDocument[]>
     getWorkspaceFolder: (uri: string) => WorkspaceFolder | null | undefined
+    getAllWorkspaceFolders: () => WorkspaceFolder[]
```

Comment:
just make it plural. helps discoverability because it will appear next to the singular case.
```suggestion
    getWorkspaceFolders: () => WorkspaceFolder[]
```